### ISSUE TITLE
Revealing Secret Room in POST_NEW_LEVEL does not work

### ISSUE THREAD

**Issue by budjmt** [TazTxUK/MinimapAPI#31]
I have the following code:
```lua
-- Reveal the secret room on the map
mod:AddCallback(ModCallbacks.MC_POST_NEW_LEVEL, function()
    local level = game:GetLevel()
    local secret = level:QueryRoomTypeIndex(RoomType.ROOM_SECRET, false, grng)
    if secret and secret >= 0 then
        local desc = level:GetRoomByIdx(secret)
        desc.DisplayFlags = desc.DisplayFlags | 5
        level:UpdateVisibility()
    end
end)
```
I have minimap api enabled (my mod name starts with D) and the secret room does not appear on the map. Without it, it does.

**Comment by TazTxUK** [comment]
Fixed as of commit 0291bb785eee61ac9f2739b7c1c0ea2943e9041e

**Comment by budjmt** [comment]
Thanks for the quick turnaround!
